README.md

```diff
@@ -1,2 +1,52 @@
 # clippit
 UCB final project
+
+A mock shop called "CLIPPIT" to show our modern booking platform.
+
+Our application is design for barber shop and salons to integrate an easy-to-use scheduling platform with their websites.
+
+## Demo Application
+This app is deployed on Heroku please [Click Here](https://clippit-ucb.herokuapp.com/ "Deployed App") to access it 
+
+## Instructions for installing application
+
+* git clone [git@github.com:victorvasujoseph/clippit.git](git@github.com:victorvasujoseph/clippit.git)
+* cd clippit
+* npm install
+
+## Technologies Used
+
+* HTML
+* CSS
+* JavaScript
+* Node.js
+* React.js
+* React-Bootstrap
+* Express
+* Mongoose Mongodb
+* Moment.js
+
+
+## ScreenShots
+* Once user enter the website, our home page will show.
```

Comment:
Once a user enters...
the home page is presented similar to the following screenshot: